fallitem.py

```diff
@@ -17,13 +17,18 @@
 import pygame
 
 
-class Spike:
+class FallItem:
 
-    def __init__(self, x, y, flip, speed, path_length):
+    def __init__(self, x, y, flip, speed, path_length, _type = "spike"):
```

Comment:
`_type` is a kwarg so no need for spaces and why make it private?

Reply:
I just checked where the underscore has to be for default keywords which cannot be used as variable name. Turned it the underscore should be present after the name, I will change it to self.type_ then

Reply:
Seeing as it's not a convention we use, don't use it.

Use the typical convention we use, which is to reassign it to a private instance variable.